nntrainer/layers/acti_func.cpp

```diff
@@ -299,6 +332,47 @@ float ActiFunc::leakyReluPrime(float x) {
   return x >= 0.0f ? 1.0f : NEGATIVE_SLOPE;
 }
 
+Tensor &ActiFunc::swish(Tensor const &t_in, Tensor &t_out) {
+  unsigned int batch = t_in.batch();
+  unsigned int channel = t_in.channel();
+  unsigned int height = t_in.height();
+  unsigned int width = t_in.width();
+
+  t_in.apply([&](float x) { return sigmoid(x); }, t_out);
+  t_out.multiply_i(t_in);
+
+  return t_out;
+}
+
+Tensor &ActiFunc::swishPrime(Tensor const &t_in, Tensor const &t_out,
+                             Tensor &outgoing_derivative,
+                             Tensor const &incoming_derivative) {
+  unsigned int batch = t_in.batch();
+  unsigned int channel = t_in.channel();
+  unsigned int height = t_in.height();
+  unsigned int width = t_in.width();
+
+  if (outgoing_derivative.empty())
+    outgoing_derivative = Tensor(t_out.getDim());
+
+  for (unsigned int b = 0; b < batch; ++b) {
+    for (unsigned int c = 0; c < channel; ++c) {
+      for (unsigned int h = 0; h < height; ++h) {
+        for (unsigned int w = 0; w < width; ++w) {
+          outgoing_derivative.setValue(b, c, h, w,
+                                       t_out.getValue(b, c, h, w) +
+                                         sigmoid(t_in.getValue(b, c, h, w)) *
+                                           (1 - t_out.getValue(b, c, h, w)));
+        }
```

Comment:
I guess you can also write these function with Tensor Op.

Reply:
> I guess you can also write these function with Tensor Op. Cause, when you use the tensor op, you can get more speedup due to the blas op in Tensor.

thanks, I correct these operations!